build_keyboard.mk

```diff
@@ -13,7 +13,11 @@ endif
 include common.mk
 
 # Set the qmk cli to use
-QMK_BIN ?= qmk
+ifeq ($(shell which qmk),)
+    QMK_BIN ?= bin/qmk
+else
+    QMK_BIN ?= qmk
+endif
```

Comment:
```suggestion
QMK_BIN ?= qmk
```

Reply:
If I do that it will not build after a clean.

If when it is building, make complains once because it is still trying to run `qmk` from the path:
```
make[1]: qmk: Command not found
```

Reply:
It's also inconsistent with the behaviour in `Makefile` that automatically uses `bin/qmk` if `qmk` does not exist.